src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        int NUMBER_OF_BALLS = 3;
```

Comment:
Since it's not a constant, you can rename it just to: numberOfBalls, ballsNumber